demisto_sdk/commands/common/constants.py

```diff
@@ -221,21 +219,344 @@ def no_common_server_python(path):
+               f"available types: {[value.value for value in TypeFields]}"
+
+    @staticmethod
+    def invalid_incident_field_group_value(file_path, group):
+        return f"IF104 - {file_path}: group {group} is not a group field."
+
+    @staticmethod
+    def invalid_incident_field_cli_name_regex(file_path, cli_regex):
+        return f"IF105 - {file_path}: Field `cliName` contains non-alphanumeric letters. " \
+               f"must match regex: {cli_regex}"
+
+    @staticmethod
+    def invalid_incident_field_cli_name_value(file_path, cli_name):
+        return f"IF106 - {file_path}: cliName field can not be {cli_name} as it's a builtin key."
+
+    @staticmethod
+    def incident_field_or_type_from_version_5(file_path):
+        return f'IF107 - {file_path}: fromVersion must be at least 5.0.0'
+
+    @staticmethod
+    def invalid_incident_field_or_type_from_version(file_path):
+        return f'IF108 - {file_path}: "fromVersion" has an invalid value.'
+
+    @staticmethod
+    def new_incident_field_required(file_path):
+        return f'IF109 - {file_path}: new incident fields can not be required. change to:\nrequired: false.'
+
+    @staticmethod
+    def incident_field_type_change(file_path):
+        return f'IF111 - {file_path}: Changing incident field type is not allowed.'
+
+    @staticmethod
+    def incident_type_integer_field(file_path, field):
+        return f'IF112 - {file_path}: the field {field} needs to be a positive integer. Please add it.\n'
+
+    @staticmethod
+    def invalid_context_output(command_name, output):
+        return f'IN120 - Invalid context output for command {command_name}. Output is {output}'
+
+    @staticmethod
+    def parameter_missing_from_yml(file_path, name, correct_format):
+        return f'IN127 - {file_path}: A required parameter "{name}" is missing or malformed ' \
+               f'in the YAML file.\nThe correct format of the parameter should be as follows:\n{correct_format}'
+
+    @staticmethod
+    def parameter_missing_for_feed(file_path, name, correct_format):
+        return f'IN128 - {file_path} Feed Integration was detected A required ' \
+               f'parameter "{name}" is missing or malformed in the YAML file.\n' \
+               f'The correct format of the parameter should be as follows:\n{correct_format}'
+
+    @staticmethod
+    def no_new_release_notes(release_notes_path):
+        return F'RN101 - No new comment has been added in the release notes file: {release_notes_path}'
+
+    @staticmethod
+    def release_notes_not_formatted_correctly(release_notes_path, link_to_rn_standard):
+        return F'RN102 - File {release_notes_path} is not formatted according to ' \
+               F'release notes standards.\nFix according to {link_to_rn_standard}'
+
+    @staticmethod
+    def pack_file_does_not_exist(file_name):
+        return f'"PU100 - {file_name}" file does not exist, create one in the root of the pack'
```

Comment:
PU ? 
let's do P

Reply:
Pack Unique - I think we should stick to 2 letters for consistency.